browser/ui/views/overlay/brave_video_overlay_window_views.cc

```diff
@@ -193,7 +194,18 @@ BraveVideoOverlayWindowViews::~BraveVideoOverlayWindowViews() = default;
 
 void BraveVideoOverlayWindowViews::SetUpViews() {
   VideoOverlayWindowViews::SetUpViews();
-  UpdateControlIcons();
+  // Use CloseImageButton in order to use the same style with the close button
+  fullscreen_button_ =
+      controls_container_view_->AddChildView(std::make_unique<CloseImageButton>(
+          base::BindRepeating(&BraveVideoOverlayWindowViews::RequestFullscreen,
+                              base::Unretained(this))));
+  const std::u16string fullscreen_button_label =
+      l10n_util::GetStringUTF16(IDS_ACCNAME_FULLSCREEN);
+  fullscreen_button_->SetTooltipText(fullscreen_button_label);
+  fullscreen_button_->SetTooltipText(fullscreen_button_label);
```

Comment:
```suggestion
  fullscreen_button_->SetTooltipText(fullscreen_button_label);
```

Reply:
Oops, one of them should be `SetAccessibleName()`. Thanks